app/models/miq_queue.rb

```diff
@@ -237,7 +237,12 @@ def self.put_or_update(find_options)
     #   it's unordered nature.
     where_scope = if conds.key?(:args)
                     args = YAML.dump conds.delete(:args)
-                    MiqQueue.where(conds).where(['args = ?', args])
```

Comment:
Ok, yeah, thanks @kbrock.  It's not for this PR but yeah, it seems like we shouldn't need to compare serialized data in a query. 👍